nicegui/elements/chart.py

```diff
@@ -19,6 +92,10 @@ def __init__(self, options: Dict) -> None:
         """
         super().__init__('chart')
         self._props['options'] = options
+        urls = [f'/_nicegui/dependencies/{id}/{dependency.path.name}'
+                for id, dependency in js_extra_dependencies.items()
+                if any(e in dependency.path.name for e in extras)]
```

Comment:
@rodja Shouldn't it be

```python
if dependency.path.name in extras
```

Reply:
True. It could be simplified. See https://github.com/zauberzeug/nicegui/commit/ee4281633a026fef919f865947709df954a783d1